CHANGELOG.md

```diff
@@ -1,3 +1,113 @@
+# 100.0.0
+
+In this major release we made improvements to Ink. As well as improvements in Dialogs,
+Flexible Header, Snackbar, and TextFields.
+
+## Breaking changes
+
+The `inkColor` in `MDCInkView` is now `null_resettable`.
+
+## New features
+
+
+### Dialogs
+
+We added the ability to customize MDCAlertController presentation animation.
+
+```swift
+let alertController = MDCAlertController()
+alertController.presentationOpacityAnimationDuration = 0.5
+alertController.presentationScaleAnimationDuration = 0.25
+alertController.presentationInitialScaleFactor = 0.7
+```
+
+```objc
+MDCAlertController *alertController = [[MDCAlertController alloc] init];
+alertController.presentationOpacityAnimationDuration = 0.5;
+alertController.presentationScaleAnimationDuration = 0.25;
+alertController.presentationInitialScaleFactor = 0.7;
+```
+
+### FlexibleHeader
+
+We added the ability to infer top safe are insets from a view controller.
+
+```swift
+let flexibleHeader = MDCFlexibleHeaderViewController()
+flexibleHeader.inferTopSafeAreaInsetFromViewController = true
+```
+
+```objc
+MDCFlexibleHeaderViewController *flexibleHeader = [[MDCFlexibleHeaderViewController alloc] init];
+flexibleHeader.inferTopSafeAreaInsetFromViewController = YES;
+```
```

Comment:
`permitInferringTopSafeAreaFromTopLayoutGuideViewController` is the new API.

```
When disabled, if both inferTopSafeAreaInsetFromViewController and topLayoutGuideAdjustmentEnabled are set to YES, and the view controller selected to extract the safe area inset from (either automatically or via the delegate) is the same as topLayoutGuideViewController, the app will crash.

When enabled, the app will not crash in the situation described above. This is only supported on iOS 11+.

Enable this property before setting inferTopSafeAreaInsetFromViewController or topLayoutGuideViewController.
```

Reply:
Addressed.